lib/config/rules/react.js

```diff
@@ -34,9 +34,9 @@ module.exports = {
   // Prevent usage of deprecated methods
   'react/no-deprecated': 'error',
   // Prevent usage of setState in componentDidMount
-  'react/no-did-mount-set-state': 'error',
+  'react/no-did-mount-set-state': 'off',
   // Prevent usage of setState in componentDidUpdate
-  'react/no-did-update-set-state': 'error',
+  'react/no-did-update-set-state': 'off',
```

Comment:
In both cases there is often no alternative for updating in response to prop changes/ doing something only on the client. This will be less of an issue with the new props from state stuff, but I don't feel these are particularly bad practices.